automation/taskcluster/lib/tasks.py

```diff
@@ -409,17 +415,17 @@ def craft_nightly_signing_task(
                 'machine': {
                   'platform': 'android-all',
                 },
-                'symbol': 'Ns',
+                'symbol': '{}s'.format(capitalized_build_type[0]),
```

Comment:
If I understand correctly, `beta` would show up as `Bs`, right? I don't mean to BS you, but that symbol is already used in mozilla-central for `build-signing`. I'd prefer to not overload that symbol, if people familiar with Treeherder on m-c go to Fenix.

How about `BeS` or `beta-s` ?

Reply:
`beta-s` sounds great!